code/common/debug.cc

```diff
@@ -97,9 +100,25 @@ namespace common::debug
 
     LocalFree(symbols);
 
+  #elif defined(__linux__)
+    // Extract the backtrace to the requested depth
+    // Note that this doesn't give particularly useful results without
+    // the -rdynamic linker option
+    std::vector<void*> frame_stack;
+    ::backtrace(frame_stack.data(), depth);
```

Comment:
This will buffer overrun because you're using an empty vector.
You can use the constructor to initialize the vector with a size:
```cpp
std::vector<void*> frame_stack{ depth };
```